getting-started/kubernetes/self-managed-onprem/onpremises.md

```diff
@@ -22,28 +22,28 @@ This how-to guide uses the following {{site.prodname}} features:
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. We recommend [Customizing Calico manifests ({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes; this avoids downstream manual updates to other {{site.prodname}} resources.
+{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
```

Comment:
you cannot use {{site.prodname}} in a link....unless you know about an update that Turk made?

Reply:
Strange, I thought I had done this before.

Reply:
It seems to work in the deploy preview: https://deploy-preview-3160--calico.netlify.com/master/getting-started/kubernetes/self-managed-onprem/onpremises

Reply:
Yipee. Glad it works now!